app/views/parks/index.html.erb

```diff
@@ -1,14 +1,20 @@
-<h1> Dog Parks </h1>
```

Comment:
Curious why you removed this heading? It's your god-given right, just wondering :)

Reply:
We didn't like the way it looked and thought it was redundant